admin_manual/installation/apps_management_installation.rst

```diff
@@ -71,17 +71,60 @@ The configuration example below shows how to add a second directory, called ``ap
    :emphasize-lines: 9-13
 
 After you add a new directory configuration, you can then move apps from the original app directory to the new one. 
-To do so, follow these steps:
+If you want to identify as an example all non core / non shipped apps, use the following command:
+
+::
+
+  sudo -uwww-data ./occ app:list --shipped=false
+
+Follow these steps to manually move apps:
 
 #. :ref:`Enable maintenance mode <maintenance_commands_label>`.
 #. :ref:`Disable the apps <apps_commands_label>` that you want to move.
-#. Create a new apps directory and assign it the same user and group, and ownership permissions as the core apps directory.
+#. Create a new apps directory like ``apps-external`` and assign it the same user, group and ownership permissions as the core apps directory.
 #. Move the apps from the old apps directory to the new apps directory.
-#. Add a new app directory in ``config/config.php``.
+#. Add the key for the new app directory in ``config/config.php``.
+#. If you're using a cache, such as `Redis <../configuration/server/caching_configuration.html#clearing-the-redis-cache>`_ or `Memcached <../configuration/server/caching_configuration.html#clearing-the-memcached-cache>`_, ensure that you clear the cache.
+#. Re-enable the apps.
+#. Disable maintenance mode.
+
+Script for moving non core apps in custom app directories
+---------------------------------------------------------
+
+In the same way as you can manually move eg non core / non shipped apps to a custom directory, a script
+might automate and help in this task.
+
+Follow these steps to automate moving apps:
+
+#. :ref:`Enable maintenance mode <maintenance_commands_label>`.
+#. :ref:`Disable the apps <apps_commands_label>` that you want to move.
+#. Create a new apps directory like ``apps-external`` and assign it the same user, group and ownership permissions as the core apps directory.
+#. Run the script described below in your ownCloud directory.
+#. Add the key for the new app directory in ``config/config.php``.
 #. If you're using a cache, such as `Redis <../configuration/server/caching_configuration.html#clearing-the-redis-cache>`_ or `Memcached <../configuration/server/caching_configuration.html#clearing-the-memcached-cache>`_, ensure that you clear the cache.
 #. Re-enable the apps.
 #. Disable maintenance mode.
 
+As preparation, please install the ``jq`` library. 
+``jq`` is like sed for JSON data - you can use it to slice, filter, map and transform structured data.
```

Comment:
s/``jq`` is like/jq is like/g. As jq's already been introduced, there's no need to draw special attention to it.